src/Compilers/CSharp/Portable/Symbols/Extensions/RewrittenParameterSymbol.cs

```diff
@@ -22,8 +24,31 @@ public RewrittenParameterSymbol(ParameterSymbol originalParameter) :
 
         internal sealed override int CallerArgumentExpressionParameterIndex => _underlyingParameter.CallerArgumentExpressionParameterIndex;
 
-        internal sealed override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes => throw ExceptionUtilities.Unreachable(); // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Follow up
+        internal sealed override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes
```

Comment:
>internal sealed override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes [](http://example.com/codeflow?start=8&length=85)

I think this override belongs in `ExtensionMetadataMethodParameterSymbol`. This override should probably keep throwing and the comment should be removed.